README.md

```diff
@@ -1,15 +1,14 @@
-# Solus Frontend
+# Solus [WIP]:
+Solus is a simple and hackable place where you can store your library of retro games.
```

Comment:
This description fits better or one that resembles it:
Solus is your personal application to manage your retro games